src/shared/sleep-config.c

```diff
@@ -54,6 +54,9 @@ static void *CAPACITY_TO_PTR(int capacity) {
 }
 
 static int PTR_TO_CAPACITY(void *p) {
+        if (!p)
+                return 0; /* A missing battery has no capacity. */
```

Comment:
hmm, nah, this is taping over. 

The hashmap_get() call in get_total_suspend_interval() should never pass the NULL it got to PTR_TO_CAPACITY(), but instead just skip over the entry. 

Might make sense to make this line here into an assert() though

Reply:
get_capacity_by_name() handles this better, btw